data/json/items/gun/50.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "id": "m107a1",
+    "copy-from": "rifle_base",
+    "type": "GUN",
+    "name": "Barrett M107A1",
+    "name_plural": "Barrett M107A1",
+    "description": "A large, shoulder-fired, .50 caliber anti-materiel rifle.  Its large size, recoil, and noise is offset by its damage and range.",
+    "weight": 12602,
+    "volume": 17,
+    "price": 650000, "//" : "Minimum list price for the bolt-action, Gun Digest, 2012.",
+    "bashing": 12,
+    "to_hit": -1,
+    "material": "steel",
+    "ammo": "50",
+    "ranged_damage": -5,
+    "range": 45,
+    "sight_dispersion": 60,
+    "recoil": 50,
+    "durability": 8,
+    "barrel_length" : 5,
+    "magazine_well": 1,
+    "magazines": [ [ "50", [ "m107a1mag" ] ] ],
+    "flags": "NEVER_JAMS"
+  },
+  {
+    "id": "m2browning",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "M2 Browning HMG",
+    "name_plural": "M2 Browning HMG",
+    "description": "A heavy machine gun used by the US Military from its inception to the cataclysm, and even rarely by cataclysm survivors.  Its massive size and design make it impossible to use unless deployed or mounted to a vehicle.",
+    "weight": 38000,
+    "volume": 20,
+    "price": 1800000,
+    "bashing": 12,
+    "to_hit": -1,
+    "material": "steel",
+    "skill": "rifle",
+    "ammo": "50",
+    "range": 45,
+    "dispersion": 30,
+    "sight_dispersion": 75,
+    "aim_speed" : 5,
+    "recoil": 225,
+    "durability": 8,
+    "reload": 400,
+    "modes": [ [ "DEFAULT", "auto", 25 ] ],
+    "barrel_length" : 5,
+    "magazines": [ [ "50", [ "belt50" ] ] ],
+    "valid_mod_locations": [ [ "accessories", 4 ], [ "barrel", 1 ], [ "bore", 1 ], [ "sights", 1 ], [ "mechanism", 4 ], [ "muzzle", 1 ] ],
+    "flags": "MOUNTED_GUN"
+  },
+  {
+    "id": "m2browning_sawn",
```

Comment:
This is not correct. Remove the item completely and make an entry in `migrations.json` as set out in previous review